.gitignore

```diff
@@ -6,9 +6,6 @@ src/docs/docgen.json
 npm-debug.log*
 node_modules
 
-# Yarn
```

Comment:
As core contributors we should be doing `yarn upgrade` and be checking in lockfile changes.